rust/src/cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nmstatectl"
-version = "2.1.0"
+version = "1.3.1"
 authors = ["Gris Ge <fge@redhat.com>"]
 edition = "2018"
 
```

Comment:
I don't think this is correct.. it will cause that the nmstatectl is now rust based which is incorrect for 1.3 IMO

Reply:
The `[[bin]]` section indicate the cli is compiled to as `nmstatectl-rust` and the rpm spec still using the python CLI.

Reply:
Oh, makes sense. I missed that, thank you for explaining!